openapi.yaml

```diff
@@ -3908,6 +3908,7 @@ components:
                                         - Plain
                                         - Json
                                       x-speakeasy-param-computed: false
+                                      x-speakeasy-unknown-values: allow
```

Comment:
We did not add this - @mheap - could you confirm that this is safe? Also could you please add some description to https://github.com/Kong/platform-api/pull/1911/files - because I don't know what's the context of this change.

Reply:
Yes, this is safe

Reply:
it adds some supress plan modifier, are we 100% sure? 

https://github.com/Kong/terraform-provider-kong-mesh/pull/86/files#diff-158d69de98310ba3f85efdfadcba6d78eb9d9185f71cb9d587942202f6099566R1573